tools/StaticAnalysis/Exceptions/AzureRM.Automation/SignatureIssues.csv

```diff
@@ -59,3 +59,42 @@
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter ExcludedKbNumbers of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter IncludedKbNumbers of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter IncludedPackageClassifications of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter ExcludedPackageNameMasks of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8410","Parameter IncludedPackageNameMasks of cmdlet New-AzureRmAutomationSoftwareUpdateConfiguration does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
```

Comment:
All the entries about plurality above (excluding `Windows`) should be removed when these parameters are renamed.

Reply:
Fixed.